airflow-core/src/airflow/utils/db_manager.py

```diff
@@ -144,13 +144,14 @@ class RunDBManager(LoggingMixin):
     def __init__(self):
         super().__init__()
         self._managers: list[BaseDBManager] = []
-        managers = conf.get("database", "external_db_managers").split(",")
-
+        managers_config = conf.get("database", "external_db_managers", fallback=None)
+        if not managers_config:
+            return
+        managers = managers_config.split(",")
```

Comment:
If the config is None we do not want to stop. The auth manager can still add its DB manager

```suggestion
        if not managers_config:
            managers = []
        else:
            managers = managers_config.split(",")
```